StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/ViewControllers/PollingViewModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  PollingViewModel.swift
+//  StripePaymentSheet
+//
+//  Created by Fionn Barrett on 08/08/2023.
+//
+
+import Foundation
+@_spi(STP) import StripeCore
+@_spi(STP) import StripePayments
+@_spi(STP) import StripePaymentsUI
+@_spi(STP) import StripeUICore
+import UIKit
+
+class PollingViewModel {
+
+    let paymentMethodType: STPPaymentMethodType
+    var CTA: String
+    var deadline: Date
```

Comment:
Let's change these to computed lazy variables and remove the compute functions.

E.g.

```swift
lazy var CTA: String {
    switch paymentMethodType {
        case .UPI:
            return .Localized.open_upi_app
        default:
            // throw fatal error
   }
}
```

Can do the same for deadline.